packages/components/src/entities/SampleAliquotDetailHeader.tsx

```diff
@@ -43,13 +43,13 @@ export class SampleAliquotDetailHeader extends PureComponent<SampleAliquotDetail
             <>
                 <table className="table table-responsive table-condensed detail-component--table__fixed sample-aliquots-details-table">
                     <tbody>
-                        {this.renderDetailRow('Aliquoted from', parent, 'aliquotedfrom')}
-                        {this.renderDetailRow('Aliquoted by', createdBy, 'aliquotedby')}
-                        {this.renderDetailRow('Aliquot date', created, 'aliquoteddate')}
-                        {this.renderDetailRow('Aliquot description', description, 'aliquoteddescription')}
+                        {this.renderDetailRow('Aliquoted From', parent, 'aliquotedfrom')}
+                        {this.renderDetailRow('Aliquoted By', createdBy, 'aliquotedby')}
```

Comment:
isn't `by` typically not capitalized with title casing?

Reply:
Yes, and the same is true for "From", but the "CreatedBy" field becomes "Created By", so I'm leaving these capitalized.